src/tools/io/core.clj

```diff
@@ -168,6 +172,20 @@
                                 (str/starts-with? (.getName file) prefix))))
                  (map (fn [^File f] (.getPath f)))))))))
 
+(defmethod list-dirs :base
+  [path & [_options]]
+  (let [^File f (io/file path)]
+    (cond
+      (.isDirectory f)
+        (->> (.listFiles f)
+             (filter #(.isDirectory ^File %))
+             (map #(.getPath ^File %)))
+
+       ;; trailing slash but not a directory: nil
+       (str/ends-with? path File/separator)
+       nil
+       :else nil)))
```

Comment:
I'm not sure if those 2 last conditions provide anything. `cond` with no matching condition is `nil` anyway. You could remove those two conditions, and even replace the `cond` by `when`